arangod/ClusterEngine/ClusterIndex.cpp

```diff
@@ -421,3 +488,28 @@ ClusterIndex::coveredFields() const {
       return _fields;
   }
 }
+
+bool ClusterIndex::inProgress() const {
+  auto const& vocbase = _collection.vocbase();
+  auto const& dbname = vocbase.name();
+  auto const cid = std::to_string(_collection.id().id());
+  auto const& agencyCache =
+      vocbase.server().getFeature<ClusterFeature>().agencyCache();
+  auto [acb, idx] =
+      agencyCache.read(std::vector<std::string>{AgencyCommHelper::path(
+          "Plan/Collections/" + dbname + "/" + cid + "/indexes")});
+  auto slc = acb->slice()[0].get(std::vector<std::string>{
+      "arango", "Plan", "Collections", vocbase.name()});
+  if (slc.hasKey(std::vector<std::string>{cid, "indexes"})) {
+    slc = slc.get(std::vector<std::string>{cid, "indexes"});
+    for (auto const& index : VPackArrayIterator(slc)) {
+      if (index.get("id").copyString() == std::to_string(_iid.id())) {
+        if (index.hasKey("isBuilding")) {
+          LOG_TOPIC("fdae4", INFO, Logger::CLUSTER)  // kaveh remove
+              << " " << index.toJson();
```

Comment:
What should happen here instead of logging?

Reply:
fixed